playwright/components/card/index.ts

```diff
@@ -0,0 +1,19 @@
+import type { Page } from "@playwright/test";
+import CARD from "./locators";
+
+// component preview locators
+export const card = (page: Page) => page.locator(CARD).nth(0);
+
+export const draggableCard = (page: Page, index: number) =>
+  page
+    .locator(`[data-element="draggable-card-${index}"]`)
```

Comment:
Is it possible get the full element locators out of the `index` file and into the `locators` file (will need new locators creating obv) so the `card.pw.tsx` file then calls a neater locator? Having the full element at this level makes the `card.pw.tsx` file look messy and not as easy to read. I know this is how the previous `card.cy.tsx` file was but it would be a good opportunity to tidy that up.